day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,36 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(addtopping)
+    @toppings.push(addtopping)
+    "You added #{addtopping}"
+  end
+
+  def remove_topping(rmtopping)
+    @toppings.delete(rmtopping)
+    "You removed #{rmtopping}"
+  end
+
+  def change_protein(chprotein)
+    @protein = (chprotein)
+    "Change protein to #{chprotein}"
+  end
```

Comment:
Nice work defining your methods to change the instance variables of `@toppings` and `@protein` directly!